swimlane/core/fields/reference.py

```diff
@@ -20,29 +20,31 @@ def target_app(self):
         """Make field's target_app available on cursor"""
         return self._field.target_app
 
-    def _evaluate(self):
-        """Scan for orphaned records and retrieve any records that have not already been grabbed"""
-        retrieved_records = SortedDict()
+    def __getitem__(self, item):
+        record_id, record = [kv for kv in self._elements.items()][item]
+        if record is self._field._unset:
+            records_get = self.target_app.records.get(id=record_id)
```

Comment:
Can we duplicate try catch like in iter method so they return same log error when not found?

Reply:
for each record 1.67, includes child record creation, patching 2 records